packages/client-python/src/rocketride/core/runtime/manager.py

```
@@ -0,0 +1,261 @@
+            signal.signal(signal.SIGINT, self._signal_handler)
+
+    def _restore_signal_handlers(self) -> None:
+        """Restore original signal handlers."""
+        if self._original_sigint is not None:
+            signal.signal(signal.SIGINT, self._original_sigint)
+            self._original_sigint = None
+        if sys.platform != 'win32' and self._original_sigterm is not None:
+            signal.signal(signal.SIGTERM, self._original_sigterm)
+            self._original_sigterm = None
+
+    def _signal_handler(self, signum, frame):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_signal_handler`

Add return type annotation: `None`

(ANN202)